apps/sports-helsinki/src/domain/venue/similarVenues/__tests__/SimilarVenuesSection.test.tsx

```diff
@@ -0,0 +1,107 @@
+import type { Venue } from 'events-helsinki-components/types';
+import * as React from 'react';
+import { render, screen, waitFor } from '@/test-utils';
+import { translations } from '@/test-utils/initI18n';
+import {
+  fakeVenue,
+  fakeVenues,
+  fakeVenuesSearchList,
+} from '@/test-utils/mockDataUtils';
+import { createVenueListRequestAndResultMocks } from '@/test-utils/mocks/venueListMocks';
+import { createVenuesByIdsRequestAndResultMocks } from '@/test-utils/mocks/venuesByIdsMocks';
+import { SPORTS_DEPARTMENT_ONTOLOGY_TREE_ID } from '../../../../domain/app/appConstants';
+import getVenueSourceId from '../../../../domain/venue/utils/getVenueSourceId';
+import SimilarVenuesSection from '../SimilarVenuesSection';
+
+const ontologyWords = [
+  { label: 'Ontology 1', id: 1 },
+  { label: 'Ontology 2', id: 2 },
+  { label: 'Ontology 3', id: 3 },
+];
+const ontologyTree = [
+  { label: 'liikunta', id: SPORTS_DEPARTMENT_ONTOLOGY_TREE_ID },
+];
+const venue = fakeVenue({ ontologyWords, ontologyTree });
+
+const expectedSimilarVenues = fakeVenues(3);
+const expectedSimilarVenuesSearchList = fakeVenuesSearchList(
+  3,
+  expectedSimilarVenues
+);
+const expectedSimilarVenuesWithIds = {
+  venuesByIds: [...expectedSimilarVenues],
+};
+
+const similarVenuesQueryVariables = {
+  ontologyWordIds: venue.ontologyWords.map((o) => o?.id?.toString() ?? ''),
+  ontologyTreeIds: venue.ontologyTree.map((t) => t?.id?.toString() ?? ''),
+  first: 6,
+  orderByName: undefined,
+};
+
+const mocks = [
+  createVenueListRequestAndResultMocks({
+    variables: similarVenuesQueryVariables,
+    response: expectedSimilarVenuesSearchList,
+  }),
+  createVenuesByIdsRequestAndResultMocks({
+    variables: {
+      ids: expectedSimilarVenues.map((v) => getVenueSourceId(v.id)),
+    },
+    response: expectedSimilarVenuesWithIds,
+  }),
+];
+
+const waitForComponentToBeLoaded = async () => {
```

Comment:
I created similar kind of wait function but it's not in the main yet. This could be even more reboust with `{ timeout: 20_000 }`